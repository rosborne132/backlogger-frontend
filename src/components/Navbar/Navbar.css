nav {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  z-index: 10;
  padding: 1% 0;
  background-color: #000;
}

nav ul {
  display: flex;
  list-style: none;
}

nav ul li {
  display: inline;
  padding: 0 1%;
}

nav ul li a {
    color: #fff;
    text-decoration: none;
}

nav ul li:hover {
  cursor: pointer;
}