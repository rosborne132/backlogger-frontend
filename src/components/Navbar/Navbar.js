import React from 'react'
import { Link } from 'react-router-dom'
import './Navbar.css'

const Navbar = () => {
    return (
        <>
            <nav role="navigation">
                <ul>
                    <li><Link to ='/login'>Login</Link></li>
                    <li><Link to ='/register'>Signup</Link></li>
                </ul>
            </nav>
        </>
    )
}

export default Navbar;