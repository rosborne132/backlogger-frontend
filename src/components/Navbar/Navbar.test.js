import React from 'react'
import { shallow } from 'enzyme'
import Navbar from './Navbar'

it('shoud render without crashing', () => {
  const component = shallow(<Navbar />)
  expect(component).toMatchSnapshot()
})