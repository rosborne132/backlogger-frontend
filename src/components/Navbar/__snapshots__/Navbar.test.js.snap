// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders empty given no tabs 1`] = `ShallowWrapper {}`;

exports[`shoud render without crashing 1`] = `ShallowWrapper {}`;
